src/test/java/net/minecraftforge/test/ItemFishedTest.java

```diff
@@ -0,0 +1,42 @@
+package net.minecraftforge.test;
+
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.ItemFishedEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+@Mod(modid = "itemfishtest", name = "ItemFishTest", version = "1.0.0")
+public class ItemFishedTest
+{
+
+    private static final boolean ENABLE = false;
+    private static Logger logger;
+
+    @Mod.EventHandler
+    public void onInit(FMLPreInitializationEvent event)
+    {
+        if (ENABLE)
+        {
+            logger = event.getModLog();
+            logger.info("Enabling Fishing Test mod");
+            MinecraftForge.EVENT_BUS.register(this);
+        }
+    }
+
+    @SubscribeEvent
+    public void onItemFished(ItemFishedEvent event)
```

Comment:
This test does not cover enough. Modifying stacks is a good start, but we should also have tests here for:
* [ ] Modifying stuff related to the entity, to verify that that works or does not work
* [ ] Cancelling the event